benchmark/cluster_containers_setup.py

```diff
@@ -173,6 +208,8 @@ def __init__(self, reactor, env, options):
         self.cluster_cert = certificates_path.child(b"cluster.crt")
         self.user_cert = certificates_path.child(b"user.crt")
         self.user_key = certificates_path.child(b"user.key")
+        self._initial_num_datasets = 0
+        self._initial_num_containers = 0
```

Comment:
:question: Not particularly important in this case, but in general it seems better to avoid extra state. These two pieces of information seem like they could be looked up and then passed on to `create_datasets_and_containers` as arguments, rather than having to look them as attributes.